src/components/text-editor/link-menu/editor-link-menu.tsx

```diff
@@ -75,13 +77,26 @@ export class TextEditorLinkMenu {
         }
     }
 
+    public componentDidLoad() {
+        if (this.textInput) {
+            const inputField = this.textInput.shadowRoot.querySelector('input');
+            requestAnimationFrame(() => {
+                inputField.focus();
+            });
```

Comment:
Can't we just get rid of all this requestAnimationFrame stuff? If not, I need a very good explanation as to why we have it. :P

Reply:
This is the only place where we still need it when we want to make certain that the elements have been painted before we set the focus.